.travis.yml

```diff
@@ -111,7 +111,16 @@ env:
     - >-
       BUILD_TYPE=metadataComparisonPython
 script:
-  - src/ci/bin/test.sh
+  # print result of `uptime` command to stdout every 5 minutes in order to avoid Travis log silence timeout (10 minutes)
+  - while sleep 300; do uptime; done &
+  # execute test suite
+  - src/ci/bin/test.sh > logfile.txt
```

Comment:
It appears that this line sends stderr to the console, and captures all stdout into a file? If the test succeeds can one recover this stdout content? While the test is running can one see stuck workflow output that I believe also goes to stdout?

For example: looking at this raw log I have no clue if this passed because something was accidentally skipped or if it actually tested what it was supposed to. (False Positive vs. True Positive): https://api.travis-ci.com/v3/job/352456182/log.txt

Btw, my only reason for the `cat | grep` comment before was that I saw something that shellcheck has yelled at me about often and was passing it on as a heads up. 😉  As I'm not actively debugging Cromwell myself I'll leave it to others to decide the best interaction with CI depending on current and future requirements. In the past, when logs have overflowed Travis' defensive limits I've taken that as a signal that either/or a) I was probably spamming the logs too much, or b) too much was going on in a single Travis job. This is partially why the test matrix is now up to 31 different Travis jobs. The other reason being that Travis VMs are very lightweight and not built for running heavy JVM operations.

Reply:
Overall I agree that this is not a perfect solution, but at this point IMO it looks better than the alternatives.

>While the test is running can one see stuck workflow output that I believe also goes to stdout?

No, but in case of stuck workflow, test will fail with 3h timeout and you'll know that something went wrong and it was not a failed Centaur test. Also if the log is < 4Mb, it'll be printed out.

>For example: looking at this raw log I have no clue if this passed because something was accidentally skipped or if it actually tested what it was supposed to. (False Positive vs. True Positive): https://api.travis-ci.com/v3/job/352456182/log.txt

Yes, this would be a problem. Possible workarounds: we may similar logic for the `success` case to check size of the written log file and if it's less than 4Mb we'd `cat` it, or otherwise we don't.